query-node/mappings/src/content/commentAndReaction.ts

```diff
@@ -160,7 +160,7 @@ async function getOrCreateCommentReactionsCountByReactionId(
   return reactionsCountByReactionId
 }
 
-export function setReactionsCount(
+export function setReactionsAndRepliesCount(
```

Comment:
While it's not the scope of this PR, I've noticed that this function has a separate section for increment and decrement that are almost identical. Let's unite the code for both cases in the following fashion:
```
const change = (operation == 'INCREMENT') ? 1 : -1
...
entity.reactionsCountByReactionId += change
entity.reactionsCount += change
...
```